microblog/app/models/blog.rb

```diff
@@ -1,2 +1,6 @@
 class Blog < ApplicationRecord
+    validates :title, presence: true
+    validates :title, length: { minimum: 3, maximum: 10 }
+    validates :body, presence: true
+    validates :status, presence: true
```

Comment:
we have only publish/unpublish so you can use published instead of status and boolean instead of a string.